src/components/breadcrumb/Breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import classnames from 'classnames'
+
+export interface BreadcrumbProps {
+  label: string
+  href?: string
+  current?: boolean
+  withMetaData?: boolean
+  position?: number
+}
+
+export const Breadcrumb = (
+  props: BreadcrumbProps & JSX.IntrinsicElements['li']
+): React.ReactElement => {
+  const {
+    label,
+    href,
+    current = false,
+    withMetaData = false,
+    position = undefined,
```

Comment:
These props specifically do not need to be exposed to developers using our library. I'd like to encapsulate them as noted in my previous comment